systemverilog-plugin/UhdmAst.cc

```diff
@@ -2264,16 +2264,93 @@ void UhdmAst::process_cont_assign()
 
 void UhdmAst::process_assignment()
 {
+    const uhdm_handle *const handle = (const uhdm_handle *)obj_h;
+    const UHDM::BaseClass *const object = (const UHDM::BaseClass *)handle->object;
     auto type = vpi_get(vpiBlocking, obj_h) == 1 ? AST::AST_ASSIGN_EQ : AST::AST_ASSIGN_LE;
+    bool simple_assign = true;
+    bool shift_unsigned = false;
+    int op_type = vpi_get(vpiOpType, obj_h);
+    AST::AstNodeType node_type;
     current_node = make_ast_node(type);
+
+    if (op_type && op_type != 82) {
```

Comment:
Use the define instead of magic number.

Reply:
Done. I added my own define, as there are only defines for values 1-43 for _vpiOpType_, but in case of simple assignment we always have 82.

Reply:
In fact, there is a define for _vpiAssignmentOp_ (here: https://github.com/chipsalliance/UHDM/blob/019f05e14a018de89b8ae2f409cb1391b1f5fe91/include/sv_vpi_user.h#L390), so changed my previous modification to use this one.